common/components/controllers/CreateProjectController.jsx

```diff
@@ -35,13 +36,14 @@ class CreateProjectController extends React.PureComponent<{||},State> {
   }
   
   render(): React$Node {
-    return (
+    return !CurrentUser.isLoggedIn() ?
```

Comment:
Look at what we do for the Create Project button when users aren't logged in.  We link them to the Login page, which then redirects them to the Create Project page once they've logged in.  With your solution, the user is redirected to the Find Projects page once they've logged in, not the Create Project page.

Reply:
Alright, I'm giving it another go with the latest commit...